Assets/Plugins/README.md

```diff
@@ -1,105 +1,101 @@
 
-#### Features:
+Players on mobile interact with Mavis ID through a WebView within the game environment. Desktop users playing games through Mavis Hub see an overlay within the Mavis Hub client.
 
-- Creating a wallet without requiring web3/crypto knowledge or downloading an external app
-- Recover & access the wallet on different devices with a simple passphrase
-- Perform on-chain actions like minting NFTs, sending transactions & signing messages
-- Gas sponsoring
+## Features
 
-Integration guide: [Integrate Mavis ID into a Unity game](https://docs.skymavis.com/mavis/mavis-id/guides/integrate-unity).
+* User authorization: sign in to your app with Mavis ID.
+* Transaction sending: transfer tokens to other addresses.
+* Message signing: sign plain text messages.
+* Typed data signing: sign structured data according to the EIP-712 standard.
+* Contract calls: execute custom transactions on smart contracts.
 
-## Installation
+## Prerequisites
 
-1. Download the latest `ID.unitypackage` release from this repository.
-2. Import the file by selecting the Unity menu option **Assets > Import package > Custom Package** and importing `ID.unitypackage`.
+General requirements:
 
-## Prerequisites
+* [Unity v2020.3.48f1 or later](https://unity.com/download)
 
-### Mobile
+Desktop requirements:
 
-- Register your application with Sky Mavis to get `YOUR_APP_ID` and `YOUR_DEEP_LINK_SCHEMA`
-- Request permission to use Mavis ID
-- Go to Developer Console > your app > App Permission > Mavis ID > Request Access
+* A game distributed through [Mavis Hub](https://hub.skymavis.com).
+* [.Net v3.1](https://dotnet.microsoft.com/en-us/download/dotnet/3.1) or later.
 
-[Head to the detail guide](https://docs.skymavis.com/comming-soon) to acquired `YOUR_APP_ID` and `YOUR_DEEP_LINK_SCHEMA`
+Mobile requirements:
 
-`Android` : Settings in Unity as described in the [Customize platform-specific settings](https://docs.skymavis.com/mavis/mavis-id/guides/integrate-unity#customize-platform-specific-settings) section in the integration guide.
+* An app created in the [Developer Console](https://developers.skymavis.com/console/applications/).
+* Permission to use Mavis ID. Request in **Developer Console > your app > App Permission > Sky Mavis Account (OAuth 2.0) > Request Access**.
+* A client ID that you can find in **Developer Console > Products > ID Service > Client ID**.
+* A redirect URI registered in **Developer Console > Products > ID Service > Redirect URI**.
 
-### Initialization
+* To deploy to Android, [Android API level 24](https://developer.android.com/about/versions/nougat) or later.
+* To deploy to iOS, [iOS 12.0](https://developer.apple.com/ios/) or later.
```

Comment:
We upgraded iOS so min version from now 13.0

Reply:
Will update!